src/test/java/com/example/demo/integrationtest/LifeBalanceCheckerControllerIntegrationTest.java

```diff
@@ -0,0 +1,44 @@
+package com.example.demo.integrationtest;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class LifeBalanceCheckerControllerIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    public void whenGetRequestToService_queryParamArgumentIsValid_thenReturnTrueOrFalse() throws Exception {
```

Comment:
na szkoleniu uczymy się, że metoda powinna się od should zaczynać
test powinien składać się z given, when, then